src/pages/stacks-blockchain/overview.md

```diff
@@ -89,7 +68,26 @@ An example of this is Stacking.
 
 ### Stacking
 
-Stackers are Stacks (STX) Token holders who provide a valuable service to the network by locking up their STX for a certain
-period of time. As a reward, Stackers receive the Bitcoin that miners transfer as part of Proof of Transfer (PoX).
+Stackers are Stacks (STX) Token holders who help secure the network by effectively acting as validators on the network, incentivizing honest behavior on the part of miners. Stackers have to lock up STX for a certain period of time. As a reward, Stackers receive the Bitcoin that miners transfer as part of Proof of Transfer (PoX).
 
 -> Proof of Transfer and Stacking are in active development and are coming soon
+
+## Protocol specification
+
+Stacks improvement proposals (SIPs) are aimed at describing the implementation of the Stacks blockchain, as well as
+proposing improvements. They should contain concise technical specifications of features or standards and the rationale
+behind it. SIPs are intended to be the primary medium for proposing new features, for collecting community input on a
+system-wide issue, and for documenting design decisions.
+
+-> See [SIP 000](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-000-stacks-improvement-proposal-process.md) for more details about the SIPs process.
+
+| **Proposal**                                                                                                   | **Description**                                     |
+| -------------------------------------------------------------------------------------------------------------- | --------------------------------------------------- |
+| [SIP 001](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md)            | Burn Election                                       |
```

Comment:
We might want to think about reducing the no. of SIPs linked here. Can always include a link that lists all SIPs.

SIP-007 for example is a lot more important than others. SIP-001 "burn election" is a bit less relevant and might cause confusion between proof of burn (which Stacks 2.0 no longer uses) and PoX.

Reply:
if there is no consensus (80%) on which block should be the anchor block for the upcoming reward cycle, burn will be used, so PoB is still relevant, and SIP-001 is still relevant for determining the sortition winner for each block.  Maybe that SIP should be renamed to reflect all this?

Reply:
We should keep in mind that this particular page is the first one folks would look at when they learn about the stacks 2.0 blockchain. I think the list feels overwhelming and likely not going to engage them (how to know which of these one should look at? and why?). Removing some or changing up the order might also look confusing for first-timers. I think the best here is to simply remove the table and add a callout that all SIPs can be found in the SIP folder, more a full, in-depth understanding of each proposal